__tests__/functional/transaction-forging/bridgechain-registration.test.ts

```diff
@@ -33,8 +33,20 @@ describe("Transaction Forging - Bridgechain registration", () => {
                 .createOne();
 
             await expect(bridgechainRegistration).toBeAccepted();
+
+            const bridgechainRegistration2 = TransactionFactory.bridgechainRegistration({
```

Comment:
This should be in it's own test because the `should broadcast, accept and forge it [Signed with 1 Passphrase]` now has expectations outside of the description. Each test should do 1 thing, this test now tests if a transaction is accepted and rejected.